upgrade-notes/backend_20241025_060719.md

```diff
@@ -0,0 +1,5 @@
+#### role ROLE_ALL_API sees all company complaints ([#3534](https://github.com/shopsys/shopsys/pull/3534))
+
+-   Created relation between complaint and customer
+-   FE API: CustomerUser with role ROLE_ALL_API gets all company complaints
```

Comment:
I would not write this into upgrade notes. Generally, we do not describe the changes in upgrade notes unless the information is vital for the upgrading itself. When I picture myself in the position of a developer who upgrades the project, the only thing I need here is the link to project base diff so I see that I need to update the data fixtures. The rest can be found in the changelog or in the pull request description :wink: